README-dev.md

```diff
@@ -141,8 +141,23 @@ npm run test
 npm run test:unit
 ```
 
+#### Running Mina-Signer Tests
+`npm run build` is not recursive, thus in order for the Mina Signer tests to run you must execute the following from the root directory:
```

Comment:
```suggestion
In order for the mina-signer tests to run you must also build from inside its subdirectory:
```

Removing the header since the other kinds of tests below don't have one either